### ISSUE TITLE
Missing backgrounds in Ogre Battle 64 battles and also some cutscenes. Seems to be a known issue with many emus and many different plugins... Being the first to fix it could boost Mupen's popularity!

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#102]
Originally reported on Google Code with ID 102

```
My System: 
 - Linux distribution: Gentoo 64bit
 - Machine type: Athlon64 x2 2.5GHz
 - Mupen64Plus version: Mupen64Plus-1-3-bin-64
 - Plugins: RiceVideo 1.3 / JttL SDL 1.3 / blight SDL input 0.0.10

Summary, Game and Problem: 
 - Ogre Battle 64: Person of Lordly Caliber 
 - [RiceVideo] Found ROM 'OgreBattle64', CRC c59b41e6e31d0169-45

 - Missing/Corrupt backgrounds in battles.

Details: 
When you enter a battle, the background is all green. There are also many
cutscenes and dialogue scenes that are the same way. It seems that this is
a problem that has occurred in this game with many different N64 emulators,
including Project64. With some emus, the background is black, with others
it's green with weird spots all over. I'm not sure how hard this is to fix,
but I think this would be a great bug to hunt down, as it's one of those
titles that was a sleeper hit. In it's current state it's not unplayable or
anything, but the art of the game is really beautiful, and it's a shame to
have to play it like this.
```

Reported by `azalyn07` on 2008-06-02 09:11:52

**Comment by richard42** [comment]
```
By the way, I forgot the MD5, here it is: 
md5 code: F666E020218392E52662FDDFA1EA4F21

```

Reported by `azalyn07` on 2008-06-02 09:20:16

**Comment by richard42** [comment]
```
I found some code to this effect in mudlord's upstream Rice. Maybe someone might 
want to port than work back to Mupen?


```

Reported by `sknauert@wesleyan.edu` on 2008-10-01 19:38:02

**Comment by richard42** [comment]
```
I could port the code from Mudlord's tree to ours.  Do you know which SVN versions
contain the fixes?
```

Reported by `richard42g` on 2008-10-04 20:13:18

**Comment by richard42** [comment]
```
The SVN >> http://code.google.com/p/aristole-video/ << has the lastest source code
from Mudlord Rice Video (6.1.4), but the download sections still use the older source
codes from Mudlord Rice Video ...
```

Reported by `vini.ipsmaker` on 2009-05-18 13:51:42

**Comment by richard42** [comment]
```
This video plugin (aristole-video) has changed names to 1964Video and re-focused. 
The current emutalk development thread is here:

http://www.emutalk.net/showthread.php?t=49671

This plugin was a fork or continuation of Rice Video but the developers were only
working on the D3D renderer in Rice, making all of their work Windows-only.
```

Reported by `richard42g` on 2010-01-11 03:58:09

**Comment by richard42** [comment]
```
Issue 347 has been merged into this issue.
```

Reported by `richard42g` on 2010-08-25 17:45:22

**Comment by richard42** [comment]
```
This has been fixed, thanks to a patch contributed by Bobby Smiles.  The new code decompresses
JPEG data in the RSP.  This fix will be included with the upcoming 1.99.5 release.
```

Reported by `richard42g` on 2012-02-26 03:52:48
- **Status changed**: `Fixed`
